### PR TITLE
nl-NL: Translate more strings

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

**Comment by AaronVanGeffen** [comment]
Thanks! I notice you've replaced "Chris Sawyer's Locomotion" with "OpenLoco" in a few spots as well. We should probably start doing the same thing for the English string tables, too — although I don't want it to affect the credits window yet...

**Comment by Gymnasiast** [comment]
I agree, that's why I left those alone.

Do you want me to address English in this PR as well or is it OK if I do that in another one? I have also find a few typos in the English translation that I can fix.

**Comment by AaronVanGeffen** [comment]
I think those would be more appropriately merged in a PR of their own. This one can be merged, then.

File: data/language/nl-NL.yml [hunk 1]
```diff
@@ -1111,75 +1111,75 @@ strings:
...
-  1109: "{COLOUR BLACK}Trucks:"
-  1110: "{COLOUR BLACK}Aircraft:"
-  1111: "{COLOUR BLACK}Ships:"
-  1112: "{SMALLFONT}{COLOUR BLACK}Company headquarters and details"
-  1113: "{SMALLFONT}{COLOUR BLACK}Company owner and status"
-  1114: "{SMALLFONT}{COLOUR BLACK}Company finances"
+  1101: "{COLOUR WINDOW_2}Hoofdkleurstelling:"
+  1102: "{COLOUR BLACK}Stoomlocomotieven:"
+  1103: "{COLOUR BLACK}Diesellocomotieven:"
+  1104: "{COLOUR BLACK}Electrische locomotieven:"
```
Comment (marijnvdwerf) [inline data/language/nl-NL.yml]: Has been spelt with a k since 1995 ;p

File: data/language/nl-NL.yml [hunk 2]
```diff
@@ -1111,75 +1111,75 @@ strings:
...
   1115: "{SMALLFONT}{COLOUR BLACK}Cargo delivered"
-  1116: "{SMALLFONT}{COLOUR BLACK}Company colour scheme"
+  1116: "{SMALLFONT}{COLOUR BLACK}Kleurstelling bedrijf"
   1117: "{SMALLFONT}{COLOUR BLACK}Company challenge for this game"
-  1118: "{COLOUR WINDOW_2}Special colour schemes used for:"
-  1119: "{SMALLFONT}{COLOUR BLACK}Select main colour"
-  1120: "{SMALLFONT}{COLOUR BLACK}Select secondary colour (used only on some vehicle types)"
+  1118: "{COLOUR WINDOW_2}Gebruik speciale kleurstellingen voor:"
+  1119: "{SMALLFONT}{COLOUR BLACK}Selecteer hoofdkleur"
+  1120: "{SMALLFONT}{COLOUR BLACK}Selecteer secondaire kleur (wordt niet op alle voertuigen gebruikt)"
```
Comment (marijnvdwerf) [inline data/language/nl-NL.yml]: Secundair lijkt gebruikelijker te zijn https://www.schrijfwijzer.nl/verwarwoordenboek/verwarwoord/473/secondair-secundair https://taal.vrt.be/secundair-secondair

File: data/language/nl-NL.yml (hunk 1 above)
Comment (Gymnasiast) [reply data/language/nl-NL.yml]: Klopt ja, zat niet op te letten bij het aanpassen van de string.

**Pull request merged**